apps/definition_accept/test/accept_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule AcceptTest do
```

Comment:
Nice work TDDing this and making nice, legible tests!

Reply:
Thanks!